dts/bindings/gpio/rohm,bd8lb600fs.yaml

```diff
@@ -18,7 +20,6 @@ properties:
   ngpios:
     type: int
     required: true
-    const: 8
```

Comment:
Perhaps default to 8 for the simple, non-chained use case?

Reply:
good idea

Reply:
done